src/pages/cart/components/CartCard.jsx

```diff
@@ -28,6 +33,30 @@ export const CartCard = ({ item }) => {
     }
   };
 
+  const addProductToWishlist = async () => {
+    try {
+      const { status, data } = await addToWishlist(auth.encodedToken, item);
+      if (status !== 201) return;
+      setWishlist(data.wishlist);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const removeProductFromWishlist = async () => {
+    try {
+      const { status, data } = await deleteFromWishlist(
+        auth.encodedToken,
+        item._id
+      );
+      if (status !== 200) return;
+      setWishlist(data.wishlist);
+      return;
+    } catch (error) {
+      console.log(error);

```

Comment:
u r catching the error but just consoling it. you could show it up as an error message or just remove the console log

Reply:
Yes, that will be done in the future through alert mesaages.